application/core/plugins/Authdb/Authdb.php

```diff
@@ -125,7 +125,7 @@ public function newLoginForm()
         }
 
         $this->getEvent()->getContent($this)
-                ->addContent(CHtml::tag('span', array(), "<label for='user'>" . gT("Username") . "</label>" . CHtml::textField('user', $sUserName, array('size' => 240, 'maxlength' => 240, 'class' => "form-control"))))
+                ->addContent(CHtml::tag('span', array(), "<label for='user'>" . gT("Username") . "</label>" . CHtml::textField('user', $sUserName ? : App()->request->getPost('user'), array('size' => 240, 'maxlength' => 240, 'class' => "form-control"))))
```

Comment:
Fill $sUserName in plugin

Reply:
Password fill added

Reply:
No : fill it here : https://github.com/LimeSurvey/LimeSurvey/blob/3745f544e1dfeb01194a12cda5c3ae1481ef6e27/application/core/plugins/Authdb/Authdb.php#L118